src/lib.rs

```diff
@@ -441,7 +441,7 @@ pub enum Variant {
 )]
 #[cfg_attr(
     all(uuid_unstable, feature = "borsh"),
```

Comment:
```suggestion
    feature = "borsh",
```

The `uuid_unstable` gate should no longer be necessary now that `borsh` is stable.